controllers/eventController.js

```diff
@@ -57,6 +57,24 @@ module.exports = {
     });
   }, 
   
+  // Return events assigned to the user for the given household and populate member references
+  // match: {assigneeStatus: "claimed"},
+  //, match: {assignedStatus: "unclaimed"}
+  findUserAssignedPopulated: function(householdId,userId) {
+    return new Promise((resolve, reject) => {
+      db.Household.find({ _id: householdId },{members:0,name:0,_id:0,__v:0,invitees:0 })
```

Comment:
What are the 0's doing here?

Reply:
the 0s tell mongoose to not return the given field from the collection.  These are unnecessary fields for this particular query

Reply:
Sweet! Good to know.